tests/Microsoft.Identity.Test.LabInfrastructure/LabApiConstants.cs

```diff
@@ -37,10 +37,7 @@ public class LabApiConstants
         public const string True = "true";
         public const string False = "false";
 
-        public const string BetaEndpoint = "http://api.msidlab.com/api/userbeta";
-        //public const string LabEndpoint = "http://api.msidlab.com/api/user";
-        public const string CreateLabUser = "https://request.msidlab.com/api/CreateLabUser";
-        public const string LabEndPoint = "https://user.msidlab.com/api/user";
+        public const string LabEndPoint = "https://msidlab.com/api/user";
```

Comment:
@jennyf19 - this also "needed" to change.

Reply:
i thought i had changed that. i did in the adal one.